cypress/e2e/homepage.spec.js

```diff
@@ -0,0 +1,12 @@
+describe('Verifyinng feature in homepage', () => {
+  beforeEach(() => {
+    cy.visit('/').injectAxe();
+  });
+
+  it('user can open `/about` when click button `btnWhoIndra` in homepage', () => {
+    cy.get('[data-testid="btnWhoIndra"]').click();
+    cy.location().should(location => {
+      expect(location.pathname).to.eq('/about');
+    });
```

Comment:
// TODO
should be expect the value / data from page about to make sure this is work.